### PR TITLE
converted JavaScriptModuleRegistryTest java file to kotlin

### PR DESCRIPTION
**Pull request by stanwolverine**
## Summary:

Helping in [Kotlin-ify React Native tests](https://github.com/facebook/react-native/issues/37708), Converted [react/bridge/JavaScriptModuleRegistryTest.java](https://github.com/facebook/react-native/tree/main/packages/react-native/ReactAndroid/src/test/java/com/facebook/react/bridge/JavaScriptModuleRegistryTest.java) to kotlin file

## Changelog:

[INTERNAL] [CHANGED] - `JavaScriptModuleRegistryTest.java` file migrated from java to kotlin

## Test Plan:

Tests should pass
```
./gradlew :packages:react-native:ReactAndroid:test
```

### CODE REVIEW

**Comment by mdvacca** [comment]
Hi @stanwolverine, thanks for working on this PR 

We can't import this pull request because the author does not have a valid CLA signed. See above for instructions!

File: packages/react-native/ReactAndroid/src/test/java/com/facebook/react/bridge/JavaScriptModuleRegistryTest.kt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+package com.facebook.react.bridge
+
+import org.junit.Assert
+import org.junit.Test
+
+/**
+ * Tests for
+ *
+ * @see JavaScriptModuleRegistry
```
Comment (mdvacca) [inline packages/react-native/ReactAndroid/src/test/java/com/facebook/react/bridge/JavaScriptModuleRegistryTest.kt:16]: NIT: can you use brackets for JavaScriptModuleRegistry:


```
/**
 * Tests for [JavaScriptModuleRegistry]
 * /
```

**Comment by stanwolverine** [comment]
Hi @mdvacca , I have signed the valid CLA. It's taking time to reflect that in this PR. Maybe it will show up after sometime.

File: packages/react-native/ReactAndroid/src/test/java/com/facebook/react/bridge/JavaScriptModuleRegistryTest.kt (hunk 1 above)
Comment (stanwolverine) [reply packages/react-native/ReactAndroid/src/test/java/com/facebook/react/bridge/JavaScriptModuleRegistryTest.kt]: Done ✅

**Review by mdvacca — APPROVED** [review]
lgtm

**Pull request closed without merging**